tests/phpunit/tests/ajax/TagSearch.php

```diff
@@ -156,4 +156,40 @@ public function test_unprivileged_user() {
 		$this->_handleAjax( 'ajax-tag-search' );
 	}
 
+	/**
+	 * Test the ajax_term_search_results filter
+	 *
+	 * @ticket 55606
+	 * @covers ::ajax_term_search_results
```

Comment:
This `@covers` tag is incorrect, it should be the function name and not the filter name. There is aready a `@covers ::wp_ajax_ajax_tag_search` tag for the whole test class in the file header, so this can be removed.

Reply:
Thanks!